src/plugins/cosmos/components/modals/Send/SendCommon.ts

```diff
@@ -1,4 +1,5 @@
 export enum SendFormFields {
+  Input = 'input', // the raw field input on the address input
```

Comment:
this was required to be added to the cosmos version of the send form to satisfy the compiler, as the `Address` component is shared between them